cfgov/v1/jinja2/v1/includes/tags.html

```diff
@@ -9,31 +9,39 @@
 
    Render a tags list when given:
 
-   tags.links:    List of tags to render with:
+   tags:     List of tags to render with:
 
-       link.url:        The tag URL.
+       url:  (optional) The tag URL.
 
-       link.text:       The tag text.
+       text: The tag text.
 
-   show_heading:  The optional boolean flag to show the tags_heading element.
-                  Defaults to true.
+   show_heading: The optional boolean flag to show the tags_heading element.
+                 Defaults to true.
 
    ========================================================================== #}
 
-{% macro render(tags, show_heading=true) %}
+{% macro render(tags, show_heading=true, has_links=true) %}
```

Comment:
What is this for?

Reply:
Oops, thanks! It's cruft from an approach I abandoned. Removed in https://github.com/cfpb/consumerfinance.gov/pull/7851/commits/db07513ec56b3cc20929a355064126bf14b871ce